ibft/utils.go

```diff
@@ -0,0 +1,12 @@
+package ibft
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/bloxapp/ssv/beacon"
+)
+
+// IdentifierFormat return base format for lambda
+func IdentifierFormat(pubKey []byte, role beacon.Role) string {
```

Comment:
@nivBlox why here and not in the validator package?

Reply:
identifier is for each ibft not validator so i think it should be under ibft

Reply:
@nivBlox yeah but it's not set in the iBFT package.. I'm looking at the code and it's actually not used at all.

Reply:
@nivBlox ?